lib/livebook_web/live/layout_helpers.ex

```diff
@@ -165,22 +166,39 @@ defmodule LivebookWeb.LayoutHelpers do
   end
 
   defp sidebar_link(assigns) do
-    assigns = assign_new(assigns, :icon_style, fn -> nil end)
-
     ~H"""
     <%= live_redirect to: @to, class: "h-7 flex items-center hover:text-white #{sidebar_link_text_color(@to, @current)} border-l-4 #{sidebar_link_border_color(@to, @current)} hover:border-white" do %>
-      <.remix_icon
-        icon={@icon}
-        class="text-lg leading-6 w-[56px] flex justify-center"
-        style={@icon_style}
-      />
+      <.remix_icon icon={@icon} class="text-lg leading-6 w-[56px] flex justify-center" />
       <span class="text-sm font-medium">
         <%= @title %>
       </span>
     <% end %>
     """
   end
 
+  defp sidebar_hub_link(assigns) do
+    ~H"""
+    <%= live_redirect to: @to, class: "h-7 flex items-center hover:text-white #{sidebar_link_text_color(@to, @current)} border-l-4 #{sidebar_link_border_color(@to, @current)} hover:border-white" do %>
+      <div class="text-lg leading-6 w-[56px] flex justify-center">
+        <span class="relative">
+          <%= @hub.emoji %>
+
+          <%= if Provider.connect(@hub.provider) do %>
+            <%= if @hub.connected? do %>
+              <div class="absolute bg-green-400 w-[10px] h-[10px] border-gray-900 border-2 rounded-full right-0 bottom-0" />
+            <% else %>
+              <div class="absolute bg-red-400 w-[10px] h-[10px] border-gray-900 border-2 rounded-full right-0 bottom-0" />
```

Comment:
```suggestion
              <div class="absolute w-[10px] h-[10px] border-gray-900 border-2 rounded-full right-0 bottom-0 bg-green-400" />
            <% else %>
              <div class="absolute w-[10px] h-[10px] border-gray-900 border-2 rounded-full right-0 bottom-0 bg-red-400" />
```

so it's easier to spot the difference. Alternatively we could interpolate the class conditionally.